.github/workflows/docs.yaml

```diff
@@ -22,3 +22,29 @@ jobs:
           user_email: "github-actions@github.com"
           user_name: ${{ github.actor }}
           commit_message: ${{ github.event.head_commit.message }}
+      - name: Pushes static folder
+        uses: cpina/github-action-push-to-another-repository@main
+        env:
```

Comment:
where is the static folder? if its not there then remove it

Reply:
Removed